app/models/manual.rb

```diff
@@ -28,6 +28,10 @@ def base_path
     end
   end
 
+  def published?
+    live? || redrafted?
+  end
```

Comment:
I've brought this to Jen's attention offline. Whether the `redrafted` status will continue to exist long-term is worth discussing when @danielroseman's back. For this PR, I'd like to assume that it's going to continue existing.